Formula/bashdb.rb

```diff
@@ -57,7 +57,7 @@ def install
  bash_5_or_greater=no
  case "${bash_major}.${bash_minor}" in
 -  'OK_BASH_VERS' | '5.0')
-+  'OK_BASH_VERS' | '5.0' | '5.1')
++  'OK_BASH_VERS' | '5.0' | '5.1' | '5.2')
```

Comment:
Might be easier to do this. But maybe for 5.3...
```suggestion
+  'OK_BASH_VERS' | 5.*)
```